modules/core/src/main/java/org/apache/fluo/core/thrift/OracleService.java

```diff
@@ -26,6 +26,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@SuppressWarnings({"rawtypes", "unused", "serial", "unchecked"})
```

Comment:
I think it's okay to add for now. In the long term, it'd be nice to have a script which re-generates these files during the build when a profile is activated, and which adds these afterwards, like Accumulo does.